api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -201,15 +224,21 @@ default void listRuntimeAfterMapper(
   @Mapping(target = "dateAccessed", source = "auditInfo.dateAccessed")
   @Mapping(target = "creator", source = "auditInfo.creator")
   @Mapping(target = "appName", source = "appName")
-  @Mapping(target = "googleProject", source = "cloudContext.cloudResource")
```

Comment:
Behavior change: previously mapped all cloudContexts, now only maps GCP cloudContexts.  I don't expect this to do anything because I wouldn't expect us to be seeing any Azure responses here anyway.